File: app/src/main/java/com/jzh/parents/datamodel/data/BannerData.java
package com.jzh.parents.datamodel.data;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * 轮播图
 *
 * @author Ding
 */
public class BannerData implements Serializable {

    private static final long serialVersionUID = -8998599127251754569L;

    public String imageUrl;
    public String linkUrl;

    public BannerData(String imageUrl, String linkUrl) {

        this.imageUrl = imageUrl;
        this.linkUrl = linkUrl;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getLinkUrl() {
        return linkUrl;
    }

    public void setLinkUrl(String linkUrl) {
        this.linkUrl = linkUrl;
    }

    public static List<BannerData> parse(String jsonStr) {
        List<BannerData> adList = new ArrayList<BannerData>();
        return adList;
    }
}

File: app/src/main/java/com/jzh/parents/datamodel/data/BannerData.kt
package com.jzh.parents.datamodel.data

import com.google.gson.annotations.SerializedName

/**
 * 轮播图结构
 *
 * @author ding
 * Created by Ding on 2018/9/17.
 */
data class BannerData(@SerializedName("title") val title: String? = "",
                      @SerializedName("img_url") val imgUrl: String? = "",
                      @SerializedName("link") val linkUrl: String? = "",
                      @SerializedName("type") val type: Int = 0) {
}